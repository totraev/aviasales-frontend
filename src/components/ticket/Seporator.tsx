import React, { SFC } from 'react'
import styles from './Seporator.module.css'

import { stopsMessage } from '../../helpers/stops'

import { ISeporatorProps } from './Seporator.types'

const Seporator: SFC<ISeporatorProps> = ({ stops }) => (
  <div className={styles.seporator}>
    <div className={styles.stops}>{stopsMessage(stops)}</div>

    <div className={styles.hr}>
      <div className={styles.line} />
      <span className={styles.airplane} />
    </div>
  </div>
)

export default Seporator
