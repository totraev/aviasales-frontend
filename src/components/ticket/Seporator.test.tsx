import React from 'react';
import renderer from 'react-test-renderer';

import Seporator from './Seporator';

describe('Seporator component', () => {
  it('Should renders correctly', () => {
    const tree = renderer.create(<Seporator />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
