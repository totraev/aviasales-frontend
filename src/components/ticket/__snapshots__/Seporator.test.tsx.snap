// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seporator component Should renders correctly 1`] = `
<div
  className="seporator"
>
  <div
    className="stops"
  >
    1 ПЕРЕСАДКА
  </div>
  <div
    className="hr"
  >
    <div
      className="line"
    />
    <span
      className="airplane"
    />
  </div>
</div>
`;
